src/main/java/uk/ac/ebi/eva/contigalias/controller/AdminController.java

```diff
@@ -51,18 +51,18 @@ public AdminController(AssemblyService service) {
         this.service = service;
     }
 
-    @ApiOperation(value = "Get or fetch an assembly using its Genbank or Refseq accession.",
+    @ApiOperation(value = "Get or fetch an assembly using its GenBank or RefSeq accession.",
             notes = "Given an assembly's accession this endpoint will return an assembly that matches that accession." +
-                    " The accession can be either a genbank or refseq accession and the software will automatically " +
-                    "fetch a result from the database for any assembly having the accession as it's genbank or refseq" +
+                    " The accession can be either a GenBank or RefSeq accession and the software will automatically " +
+                    "fetch a result from the database for any assembly having the accession as its GenBank or RefSeq" +
                     " accession. This endpoint will first look for the assembly in local database and return the " +
-                    "result. If local search fails, it search for the target assembly at a remote source (NCBI by " +
+                    "result. If local search fails, it will search for the target assembly at a remote source (NCBI by " +
                     "default). If desired assembly is found at remote source, it will fetch and add it to the local " +
```

Comment:
I wasn't sure about this, but I consulted with native speakers and they agree that we should put articles here
```suggestion
                    "default). If the desired assembly is found at the remote source, it will fetch and add it to the local " +
```
this applies to other places too.

Reply:
I proofread all @ApiOperation notes and have made some changes including the ones you suggested.